### PR TITLE
Latest posts

### LINKED ISSUE #22: [Authors] Add latest articles by this author to author page
Each author page has a "Latest" section at the end, listing the most recent (or all?) articles posted by this author.

### PR DESCRIPTION
**Pull request by rofe**
Fix #22 

- [x] fetch latest posts from algolia index
- [x] render posts

### CODE REVIEW

**Comment by rofe** [comment]
todo: styling of posts

**Comment by rofe** [comment]
@kptdobe could you give it another look now?

File: head.html [hunk 1]
```diff
@@ -69,15 +93,15 @@
...
+      const r = /^By (.*)\n*(.*)$/gmi.exec(insertInside.innerText);
       const author = r && r.length > 0 ? r[1] : null;
       const date = r && r.length > 1 ? r[2] : ''
       if (author) {
         // clear the content of the div and replace by avatar and text
         insertInside.innerHTML = '';
         const xhr = new XMLHttpRequest();
         const fileName = author.replace(/\s/gm, '-').toLowerCase();
-        const pageURL = '/authors/' + fileName + '.html';
+        const pageURL = getLink(pageTypes[1], author);
```
Comment (kptdobe) [inline head.html]: `pageTypes[x]` is no so clear. Maybe we can replace the array by an object and do something like: 

`getLink(TYPES.AUTHOR, author);` 

or with single properties like:

`getLink(TYPE_AUTHOR, author);`

WDYT ?

File: head.html [hunk 2]
```diff
@@ -221,33 +248,79 @@
...
+        target.insertBefore(socialWrap, source.nextSibling);
       }
     }
   }
 
+  function fetchLatestPosts() {
+    const latestWrap = document.createElement('div');
+    latestWrap.className = 'default latest-posts';
+    getSection().parentNode.appendChild(latestWrap);
+    const searchClient = algoliasearch('LPQI0MG7ST', '9bf61456f606d21ddc1723f30500659e');
```
Comment (kptdobe) [inline head.html]: Are we supposed to expose those tokens like this ?

File: head.html (hunk 1 above)
Comment (kptdobe) [reply head.html]: Note that this would be a more global change.

File: head.html (hunk 2 above)
Comment (rofe) [reply head.html]: yes, these are fine: App ID and read only search token.

File: head.html (hunk 1 above)
Comment (rofe) [reply head.html]: Let's do that in a separate step, though.

**Pull request merged**